notifications/docker-compose.yml

```diff
@@ -0,0 +1,98 @@
+version: "3"
+services:
```

Comment:
wow! you dockerized this? How will this work on the server? @zxenonx Do we need to involve DevOps?

Reply:
@mikengr nah the dockerization instructions came from Novu. And it is meant to be hosted standalone. i.e it'll be hosted on a VM with the required ports exposed. 

Ideally, it would be hosted on a subdomain like notifications.zuri.chat with the messaging server publishing events to it while the mobile apps and frontend get a user's notification feed from the same instance.

Reply:
yeah, so how is this going to work here?

Reply:
It will be accessed via the HTTP API it provides by all the services when hosted standalone

Reply:
@mikengr I'm working with a devops to get this deployed but will need this to be merged first. Are there any blocking issues?

Reply:
#140 and #141 also will require the live link from this to work